package.json

```diff
@@ -12,15 +12,17 @@
     "serve-gmf-examples": "DEV_SERVER=1 TARGET=gmf-examples webpack serve --open --server-type=https --server-options-cert=private.crt --server-options-key=private.key --port=3000 --mode=development --progress --watch",
     "serve-gmf-apps": "DEV_SERVER=1 TARGET=gmf-apps webpack serve --open --server-type=https --server-options-cert=private.crt --server-options-key=private.key --port=3000 --mode=development --progress",
     "serve-api": "webpack serve --content-base=api/dist/apihelp/ --config=buildtools/webpack.api.js --server-type=https --server-options-cert=private.crt --server-options-key=private.key --port=3000 --mode=development --progress --watch",
+    "build-gmf-apps-profile": "TARGET=gmf-apps webpack --profile --json > profile.json",
     "dist": "TARGET=dist webpack --mode=production --progress",
     "dist-spinner": "svgo --config=svgo.js src/icons/spinner.svg --output=dist/spinner.svg",
-    "build-gmf-apps-profile": "TARGET=gmf-apps webpack --profile --json > profile.json",
-    "prepare": "tsc --pretty",
+    "distlib": "tsc --pretty",
+    "distlib-scss": "copyfiles src/**.scss distlib",
```

Comment:
I was wondering if we wanted to do that, but it's already in sources so I've done nothing.
I think that's a good thing, but sometimes, like in OL, they prefers to provide only one big `.css` file in the "dist" folder.
That's just for the notice. For me it's ok (and good) for ngeo to provide css like you have done here.

Reply:
The SCSS is not easy usable because for that, we need a different alias for JS and CSS...
The all-in-one CSS is available in the dist folder :-)